includes/models/model.llms.instructor.php

```diff
@@ -125,35 +126,60 @@ public function get_memberships( $args = array(), $return = 'llms_posts' ) {
 	}
 
 	/**
-	 * Retrieve instructor's posts (courses and memberships, mixed)
+	 * Retrieve instructor's posts (courses and memberships, mixed).
 	 *
-	 * @param    array  $args    query arguments passed to WP_Query
-	 * @param    string $return  return format [llms_posts|ids|posts|query]
-	 * @return   mixed
-	 * @since    3.13.0
-	 * @version  3.13.0
+	 * @since 3.13.0
+	 * @since [version] Added `$include_assistant` parameter.
+	 *
+	 * @param array   $args              Query arguments passed to WP_Query.
+	 * @param string  $return            Return format [llms_posts|ids|posts|query].
+	 * @param boolean $include_assistant Include posts whose instructor is just a parent of this instructor.
+	 * @return mixed
 	 */
-	public function get_posts( $args = array(), $return = 'llms_posts' ) {
+	public function get_posts( $args = array(), $return = 'llms_posts', $include_assistant = false ) {
 
-		$serialized_id = serialize( // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.serialize_serialize
+		$parent_instructors = $include_assistant ? $this->get( 'parent_instructors' ) : array();
+		$instructor_ids     = array_merge(
+			$parent_instructors,
```

Comment:
Reduce the number of lines it takes to add a single array element.
```suggestion
		$instructor_ids = $include_assistant ? $this->get( 'parent_instructors' ) : array();
		$instructor_ids[] = $this->get_id();
```
GitHub wouldn't let me suggest with all the lines used in `array_merge()`.

Reply:
weird github.
Thanks for the suggestion!